whipper/command/cd.py

```diff
@@ -341,7 +350,19 @@ def doCommand(self):
         else:
             logger.info("creating output directory %s", dirname)
             os.makedirs(dirname)
-
+        self.coverArtPath = None
+        if (self.options.fetch_cover_art in {"embed", "complete"} and
+                importlib.util.find_spec("PIL") is None):
+            logger.warning("the cover art option '%s' won't be honored "
+                           "because the 'pillow' module isn't available",
+                           self.options.fetch_cover_art)
+        elif self.options.fetch_cover_art in {"file", "embed", "complete"}:
+            self.coverArtPath = self.program.getCoverArt(
+                                    dirname,
+                                    self.options.fetch_cover_art,
+                                    self.program.metadata.mbid)
+        if self.options.fetch_cover_art == "file":
+            self.coverArtPath = None  # This way the image won't be embedded
         # FIXME: turn this into a method
```

Comment:
What is this comment for? The code above or below it? It seems a bit weird to have it coupled this closely at the end of this chunk.

Reply:
@Freso , do you think this is considered resolved now?

Reply:
This is an old relic from morituri's era. Fixed it in commit fb9fb34b83bf17804b265a42dd562aa73022f55e.